hack/test_errata.py

```diff
@@ -0,0 +1,598 @@
+
+
+@patch("urllib.request.urlopen")
+def test_notify_format_of_message_pr_approved(urlopen_mock):
+    message = {
+        "fulladvisory": "RHEL",
+        "when": "today",
+        "synopsis": "OpenShift Container Platform X.X.X",
+        "uri": "example_uri",
+        "approved_pr": "example_approved"
+    }
+
+    errata.notify(message, True)
+
+    msg_text = '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis} {uri}'.format(**message)
+    msg_text += "\nPR {approved_pr} has been approved".format(**message)
+    
+    expected_data = urllib.parse.urlencode({
+        'payload': {
+            'text': msg_text,
+        },
+    }).encode('utf-8')
+    assert urlopen_mock.call_args[1]["data"] == expected_data
+
+
+# Mocking classes of PyGithub
+GitUser = namedtuple("GitUser", "login")
+GitLabel = namedtuple("GitLabel", "name")
+
+class GithubRepoPrMock:
+    def __init__(self, user, title, labels=[], number=0, body="", url="", html_url=""):
+        self.user = user
+        self.title = title
+        self.labels = labels
+        self.number = number
+        self.body = body
+        self.url = url
+        self.html_url = html_url
+        self.create_issue_comment = MagicMock()
+
+    def __eq__(self, other):
+        if not isinstance(other, GithubRepoPrMock):
+            return False
+        return      self.user == other.user     \
+                and self.title == other.title   \
+                and self.labels == other.labels \
+                and self.number == other.number \
+                and self.body == other.body     \
+                and self.url == other.url       \
+                and self.html_url == other.html_url
+
+
+def test_get_open_prs_to_fast_labels_not_lgtmed():
+    # Create multiple groups of label(s)
+    labels_empty = []
+    labels_bug = [GitLabel("bug")]
+    labels_duplicate = [GitLabel("duplicate")]
+    labels_enhancement = [GitLabel("enhancement")]
+    labels_multiple = [GitLabel("bug"), GitLabel("duplicate"), GitLabel("documentation")]
+
+    # Mock Pullrequests in a repository
+    prs = []
+    prs.append(GithubRepoPrMock(GitUser("openshift-bot"), "Enable ABC in fast channel(s)", labels_empty, 1))
+    prs.append(GithubRepoPrMock(GitUser("openshift-bot"), "Enable GHI in fast channel(s)", labels_bug, 2))
+    prs.append(GithubRepoPrMock(GitUser("openshift-bot"), "Enable JKL in fast channel(s)", labels_duplicate, 3))
+    prs.append(GithubRepoPrMock(GitUser("openshift-bot"), "Enable DEF in fast channel(s)", labels_enhancement, 4))
+    prs.append(GithubRepoPrMock(GitUser("openshift-bot"), "Enable MNO in fast channel(s)", labels_multiple, 5))
+
+    # Mock repo.get_pulls used in errata.get_open_prs_to_fast
+    repo = MagicMock()
+    repo.get_pulls = MagicMock(return_value=prs)
+
+    get_prs = []
+    for pr in errata.get_open_prs_to_fast(repo):
+        get_prs.append(pr)
+
+    assert get_prs == prs
+
+
+def test_get_open_prs_to_fast_labels_lgtmed():
```

Comment:
Not quite following what does it do - this checks that lgtm'ed PRs are not listed in `get_open_prs_to_fast`?